lib/config/validation.spec.ts

```diff
@@ -1,8 +1,11 @@
+import { getDatasourceList } from '../modules/datasource';
 import { configFileNames } from './app-strings';
 import { GlobalConfig } from './global';
 import type { RenovateConfig } from './types';
 import * as configValidation from './validation';
 
+const datasourceList = getDatasourceList();
```

Comment:
can we mock datasources? (perf)

if possible also mock managers. 🤗

Reply:
How to do this?
Do I mock the modules like we have in other test files?

Reply:
yes

Reply:
why managers>

Reply:
Can't mock the managers module. Other imports in the validation file depend on it and it breaks them.